dbschema/tool.gel

```diff
@@ -0,0 +1,14 @@
+module default {
+  type Tool extending Resource, Mixin::Named, Mixin::Timestamped {
```

Comment:
Resource is already timestamped so don't need that. name needs to have its own unique constraint, as all _tool names_ need to be unique.
```suggestion
  type Tool extending Resource, Mixin::Named {
      overloaded name {
        constraint exclusive;
      }
```